src/GL/rsp_gl.S

```diff
@@ -631,6 +650,11 @@ GLCmd_PreInitPipe:
 1:
     or modes1, t3
 
+    # * Skip if FLAG2_USE_RDPQ_MATERIAL is active
+    andi t1, state_flags2, FLAG2_USE_RDPQ_MATERIAL
+    bgtz t1, rdpq_skipcombinerblender
+    nop
```

Comment:
To stay consistent with the rest of the code I'd use `bnez` instead of `bgtz`, which is guaranteed to ignore the sign bit (just to be safe). Also, the `nop` is not necessary here and I adjusted the label name according to my suggestion below.
```suggestion
    bnez t1, gl_skipcombinerblender
```